packages/admin/src/pages/dashboard/ManageGrantSchemes.tsx

```diff
@@ -1,42 +1,78 @@
+import moment from 'moment';
 import CustomLink from '../../components/custom-link/CustomLink';
 import Table from '../../components/table/Table';
 import Scheme from '../../types/Scheme';
-import { generateSchemeTableRows } from '../scheme-list/index.page';
 
-const ManageGrantSchemes = ({ schemes }: ManageGrantSchemesProps) => {
+export const generateSchemeTableRows = ({ schemes }: SchemesProps) => {
+  return schemes.map((scheme) => {
+    const schemeLink = (
+      <CustomLink
+        href={`/scheme/${scheme.schemeId}`}
+        dataCy={`cy_linkToScheme_${scheme.name}`}
+        ariaLabel={`View scheme ${scheme.name}`}
+      >
+        {scheme.name}
+      </CustomLink>
+    );
+
+    const createdDate = scheme.createdDate
+      ? dateFormatter(scheme.createdDate)
+      : '-';
+
+    const updatedDate = scheme.lastUpdatedDate
+      ? dateFormatter(scheme.lastUpdatedDate)
+      : '-';
+
+    const lastUpdatedBy = scheme.lastUpdatedBy ? scheme.lastUpdatedBy : '-';
+
+    return {
+      cells: [
+        {
+          content: schemeLink,
+        },
+        { content: createdDate },
+        { content: updatedDate },
+        { content: lastUpdatedBy },
+      ],
+    };
+  });
+};
+
+export function dateFormatter(date: string) {
+  const utcDate = moment(date).utc();
+  return moment(utcDate).local().format('D MMMM YYYY, h:mm a');
```

Comment:
Think we're dropping the space here going forward - Matt corrected me on this yesterday

```suggestion
  return moment(utcDate).local().format('D MMMM YYYY, h:mma');
```

Reply:
cheers John